playbook.md

```diff
@@ -1832,6 +1832,18 @@ Mobile - if you’re comfortable to share
 www.dxw.com :: 0345 2577520 :: @dxw helping public sector teams to make good
 technology decisions and build better services
 
+### Out of office email message
+
+If you require an out of office message on your email account then you will
+need to send your request to support@dxw.com with the start and return date
+for this to be added to our mail server.
```

Comment:
I would also suggest adding "and the contact details for another member of the team for any urgent queries" after "return date".

Reply:
I've amended to include your suggested wording, please review.